packages/react-components/src/queries/useHistoricalAssetPropertyValues/useRefreshRate.ts

```diff
@@ -29,11 +32,13 @@ export const useRefreshRate = ({
   refreshRate,
   viewport,
   minimumRefreshRate = 5000,
```

Comment:
Can the minimum refresh rate be smaller? We'll probably have to stress test this

Reply:
It can be smaller, the minimum in general we support is 1s. I can make it that.